Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -293,12 +338,31 @@ void IMixedRealitySpeechHandler.OnSpeechKeywordRecognized(SpeechEventData eventD
                     {
                         if (eventData.MixedRealityInputAction == voiceSelect[i])
                         {
+                            EyeTrackingTarget.SelectedTarget = this.gameObject;
                             OnSelected.Invoke();
                         }
                     }
                 }
             }
         }
         #endregion
+
+        #region Methods to Invoke Events Manually
+        public void RaiseSelectEventManually()
+        {
+            EyeTrackingTarget.SelectedTarget = this.gameObject;
+            OnSelected.Invoke();
+        }
+
+        public void RaiseEventManually_TapDown()
+        {
+            OnTapDown.Invoke();
```

Comment:
It looks like `OnTapDown` is public. Can you call `.Invoke()` on that directly instead of wrapping it in a new public method?